src/components/inspect-panel-info.animal.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from "react";
+import { BackpackMouseType, InspectContext } from "../models/backpack-mouse";
+import { speciesDef } from "../models/units";
+import { renderGameteIcons } from "./inspect-panel";
+
+export function renderAnimalInfo(
+    org: BackpackMouseType, context: InspectContext, isGamete: boolean, showGenotype: boolean): JSX.Element {
+  const species = speciesDef(org.species);
+  const sexLabel = org.sex === "female" ? "Female" : "Male";
+  const colorLabel = species.getPhenotypeLabel(org.phenotype);
+  const genotypeLabel = species.getGenotypeHTMLLabel(org.genotype);
+  const rowClass = "info-row" + (context === "nest" ? "" : " wide");
+  const mouseInfoClass = "mouse-info" + (context === "population" ? " no-header" : "");
+  const infoTypeClass = "info-type" + (context === "population"  ? " population " : "");
+  return (
+    <>
+      <div className="mouse-label">
+        {context === "offspring" ? "Offspring" : (org.sex === "female" ? "Mother" : "Father")}
+      </div>
+      <div className={mouseInfoClass}>
+        <div className={rowClass}>
+          <div className={infoTypeClass}>
+            {speciesDef(org.species).phenotypeHeading + ": "}
+            <span className="info-data">{colorLabel}</span>
+          </div>
+        </div>
+        { (!isGamete || context === "offspring") &&
+          <div className={rowClass}>
+            <div className={infoTypeClass}>
+              {"Sex: "}
+              <span className="info-data">{sexLabel}</span>
+            </div>
+          </div>
+        }
+        { showGenotype &&
+          <div className={rowClass}>
+            <div className={infoTypeClass}>
+              {"Genotype: "}
+              <span className="info-data"
+                dangerouslySetInnerHTML={{ __html: genotypeLabel }}
```

Comment:
FYI, in other projects we've used a `renderHtml()` utility function (implementation below) to avoid having to `dangerouslySetInnerHTML`:
```typescript
import DOMPurify from "dompurify";
import parse from "html-react-parser";

// from module htmlparser2
interface DomElement {
    attribs?: {[s: string]: string};
    children?: DomElement[];
    data?: any;
    name?: string;
    next?: DomElement;
    parent?: DomElement;
    prev?: DomElement;
    type?: string;
}
export type ParseHTMLReplacer = (domNode: DomElement) => JSX.Element | void | undefined | null | false;

export function renderHTML(html: string, replace?: ParseHTMLReplacer) {
  return parse(DOMPurify.sanitize(html || ""), { replace });
}
```

Reply:
I can see the point of this. In this case we're passing around HTML from within the app, but that of course highlights that we simply shouldn't be doing that, we should just be passing around JSX if needed.

At this point, since that same label is not just used in that view but passed in as props in a number of other views unconnected to this work, I'm going to leave this, but it's a good thing to fix.